core/src/test/java/dev/streamx/cli/license/LicenseFetcherMalformedTest.java

```diff
@@ -5,14 +5,13 @@
 
 import dev.streamx.cli.exception.LicenseException;
 import dev.streamx.cli.license.LicenseWiremockConfigs.MalformedWiremockLicense;
-import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.common.WithTestResource;
 import io.quarkus.test.junit.QuarkusTest;
 import jakarta.inject.Inject;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
-@QuarkusTestResource(restrictToAnnotatedClass = true,
-    value = MalformedWiremockLicense.class)
+@WithTestResource(MalformedWiremockLicense.class)
```

Comment:
1. `@QuarkusTestResource` become deprecated
2. `@WithTestResource` is the replacement suggested in deprecation comment
3. `restrictToAnnotatedClass = true` is now the default value in `WithTestResource`
4. The remaining field `value` is now the only required setting, so "value = " can be skipped